src/MonthList/index.js

```diff
@@ -220,14 +220,9 @@ export default class MonthList extends Component {
   };
 
   render() {
-    let {
```

Comment:
Have you run eslint against these files?

These changes do not look compatible with our prettier settings

Reply:
ah yes - I have some trouble getting vscode to correctly inherit repo eslint settings over my default configurations.  I'll revert the formatting settings in the next few days

Reply:
you can run it with cli 
```bash
yarn eslint --fix .
```